### DISCUSSION TITLE
Update EAF apps after a theme change in Emacs

### DISCUSSION THREAD

**Discussion by Inc0n** [emacs-eaf/emacs-application-framework#980] [Q&A]
I noticed how my music app doesn't update theme immediately until a new instance is fired to the current emacs theme.

What sort of functions is there available to make this happen?

Thanks

**Comment by manateelazycat** [comment]
EAF can't not change color with emacs theme, it's not easy to do.

**Reply by Inc0n** [reply]
Thanks, I see now.

Would I be right to think that this requires each app to expose a function that allows flexible theme change?

**Reply by manateelazycat** [reply]
It is not unable to provide API do that, but that some applications cannot switch themes dynamically. this need change a large number of code, not a relaxed job.

**Reply by Inc0n** [reply]
That's unfortunate to hear, thanks for answering my question.
